.gitignore

```diff
@@ -107,7 +107,6 @@ _artifacts
 out
 
 # github.com
-github.com
 profile.cov
 coverage.xml
```

Comment:
Let's drop all of these, and also the two .env/.envrc below, we need them for our build system Konflux